README.md

```diff
@@ -8,23 +8,59 @@ Alpakka [![scaladex-badge][]][scaladex] [![travis-badge][]][travis] [![gitter-ba
 [gitter]:                    https://gitter.im/akka/akka
 [gitter-badge]:       https://badges.gitter.im/akka/akka.svg
 
-This project provides a home to Akka Streams connectors to various technologies, protocols or libraries.
+Systems don't come alone. In the world of microservices, cloud deployment and services, and our history of existing solutions we need to integrate. Reactive Streams give us a technology-independent tool to let these systems communicate without overwhelming each other.
+
+Alpakka's connectors let you stream data in and out of other technologies using Akka Streams. 
+
+[Akka Streams](https://doc.akka.io/docs/akka/current/stream/index.html) is a [Reactive Streams](http://www.reactive-streams.org/) and JDK 9+ [java.util.concurrent.Flow](https://docs.oracle.com/javase/10/docs/api/java/util/concurrent/Flow.html)-compliant implementation and therefore [fully inter-operable](https://doc.akka.io/docs/akka/current/general/stream/stream-design.html#interoperation-with-other-reactive-streams-implementations) with other implementations. 
+
 
 Documentation
 -------------
 
-See our [documentation](http://developer.lightbend.com/docs/alpakka/current/) and [scaladoc](http://developer.lightbend.com/docs/api/alpakka/current/akka/stream/alpakka/index.html) pages.
+- [Alpakka reference](http://developer.lightbend.com/docs/alpakka/current/) documentation 
+
+- [Alpakka Kafka connector](https://doc.akka.io/docs/akka-stream-kafka/current/) documentation
```

Comment:
include documentation in the link?

Reply:
Alpakka Kafka connector is not really a word we use anywhere; Reactive Kafka or Alpakka Kafka?

Reply:
Add `reference` also to the Alpakka Kafka connector link? Just to keep the symmetry. :)